### ISSUE TITLE
Alt tags

### ISSUE THREAD

**Issue by mml1** [eunhyulkim/capstone_colmar#3]
(Subtlety) Be sure to write a short description of what is in your image in your alt tag for all your images. Alt tags are used when your image fails to load and by screen readers. People who have low vision or are blind use screen readers to tell them what the page holds. A screen reader will read what is in the alt tag to give your user information about the image. Please spend the time adding content in your alt tags. 

`<img src="resources/images/course-computer-art.jpg" alt="">
`
https://github.com/eunhyulkim/capstone_colmar/blob/master/index.html#L122
